src/squeue.rs

```diff
@@ -142,6 +143,20 @@ impl Inner {
     }
 }
 
+impl Debug for Inner {
```

Comment:
I think users should access values through methods such as `.len()`, not through Debug.

Reply:
This impl is necessary so that we can `#[derive(Debug)]` on the `IoUring` itself. Are you saying I should instead print out the length of the squeue in the `Debug` impl?

Reply:
Is there any practical use case need to print IoUring?

Reply:
Well, debugging. There are all sorts of bugs that can occur with an API like this, and we never know what information users might need to help resolve them. Also, there is a [guideline that everything must implement Debug](https://rust-lang.github.io/api-guidelines/debuggability.html#all-public-types-implement-debug-c-debug).

Reply:
The user can manually access any useful information without going through the Debug method.

In production, I encountered a lot of problems caused by abuse of Debug, so I was cautious about implementing Debug.

Reply:
That's true, but `dbg!(&uring);` is a lot easier than having to know the fields individually - especially when debugging we want to make it easy. Problems by abusing `Debug` can be avoided with the [`clippy::use_debug`](https://rust-lang.github.io/rust-clippy/master/index.html#use_debug) lint, so while it does require users to manually add that it is something they should be doing in production.